process/economicMetrics.go

```diff
@@ -0,0 +1,70 @@
+package process
+
+import (
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-proxy-go/data"
+)
+
+// EconomicsDataPath represents the path where an observer exposes his economics data
+const EconomicsDataPath = "/network/economics"
+
+// GetEconomicsDataMetrics will return the economic metrics from cache
+func (nsp *NodeStatusProcessor) GetEconomicsDataMetrics() (*data.GenericAPIResponse, error) {
+	economicMetricsToReturn, err := nsp.economicMetricsCacher.Load()
+	if err == nil {
+		return economicMetricsToReturn, nil
+	}
+
+	log.Info("economic metrics: cannot get from cache. Will fetch from API", "error", err.Error())
+
+	return nsp.getEconomicsDataMetricsFromApi()
```

Comment:
we might remove this extra call + the info on line 20 and directly return error here

Reply:
done